jte-gradle-plugin/build.gradle

```diff
@@ -6,8 +6,8 @@ plugins {
 }
 
 repositories {
-    mavenCentral()
     mavenLocal()
+    mavenCentral()
```

Comment:
Moving it down here makes the plugin search for the snapshots in the local cache first, where they probably are.